### ISSUE TITLE
It wont let me flash win 10 iso on usb

### ISSUE THREAD

**Issue by aidenhyde** [slacka/WoeUSB#254]
## Good H
![Screenshot from 2019-04-06 11-48-15](https://user-images.githubusercontent.com/43505661/55672695-7b96d580-5863-11e9-891d-d1194a073403.png)
abit Checklist for Issue Reporting
Checkout [Mastering Markdown · GitHub Guides](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown) if you need help on the GitHub-flavored Markdown syntax.  You may remove sections and contents that don't apply.

* [ ] I've searched the issue tracker and is pretty sure that there's no duplicate issue already filed
* [ ] I've built the latest development snapshot using the instructions in README and verified that the issue can still be reproduced (for bug reports)

## Issue Reproduce Instructions
> 1. Launch WoeUSB by clicking on the icon
> 2. selct iso then it happens on Ubuntu 18.04.2

## Expected Behavior
> No error

## Current Behavior
> WoeUSB errors with message ""Error:File "/media/woeusb_source_1554569285_17755/source/install.wim" in source image has exceed the FAT32 filesystem 4gib single file size limitation and cannot be installed"

### WoeUSB Version
> For source code build, run `git describe --tags --always --dirty` under the source tree, for prebuilt releases refer the source that provides WoeUSB

### WoeUSB Source
> Built from source code, commit abcdefg
> Installed from Ubuntu 16.04 software archive
> Installed from WebUpd8 PPA

### GNU Bash Version
> WoeUSB exploits several advanced features of the Bash scripting language and requires a relatively-recent Bash intepreter.  Run `bash --version` in a terminal to acquire this information.

### Information about the Operating System
> Different OS distributions provides different characteristics and may influence the result of running WoeUSB.  Run `lsb_release --description` or `lsb_release -d` in a terminal to acquire this information.

### Information about the Source Media
> "Windows 9 Professional Edition" downloaded from <https://microsoft.com/download/windows-9>

### Information about the Target Device
> Transcend JetFlash® 790K 64GB (USB 3.1 variant)

**Comment by themainliner** [comment]
I can confirm that the version from the Ubuntu PPA (Version @@WOEUSB_VERSION@@ - nice) is similarly borked. I made sure to manually unmount my USB drive and ensured it was formatted as NTFS. Win10 contains install.win which at 4.6 GB will not copy to FAT32 partition, which is limited to a 4GB file size. I succeed by using the command line, the GUI is indeed not working correctly.

Command line:  
woeusb --device <source media path> <device>
sudo woeusb --target-filesystem NTFS --device Win10_*.iso /dev/sdx
